client/components/chart/utils.js

```diff
@@ -511,14 +522,31 @@ export const drawLines = ( node, data, params ) => {
 		.append( 'g' )
 		.attr( 'class', 'focus' );
 
-	focus
-		.append( 'line' )
+	const focusGrid = focus
+		.append( 'g' )
 		.attr( 'class', 'focus-grid' )
+		.attr( 'opacity', '0' );
+
+	focusGrid
+		.append( 'line' )
 		.attr( 'x1', d => params.xLineScale( new Date( d.date ) ) )
 		.attr( 'y1', 0 )
 		.attr( 'x2', d => params.xLineScale( new Date( d.date ) ) )
 		.attr( 'y2', params.height )
-		.attr( 'opacity', '0' );
+		.style( 'opacity', 1 );
```

Comment:
There is no need to specify the opacity here, right? By default it's always `1`.